packages/patternfly-4/react-core/src/components/Wizard/WizardHeader.tsx

```diff
@@ -5,25 +5,31 @@ import { Button } from '../Button';
 import { Title } from '../Title';
 import { TimesIcon } from '@patternfly/react-icons';
 
-interface WizardHeaderProps {
+export interface WizardHeaderProps {
+  /** Callback function called when the X (Close) button is clicked */
   onClose?: () => void;
+  /** Title of the wizard */
   title: string;
-  description: string;
-  ariaLabel: string;
```

Comment:
this would be a breaking change for the consumers already using the wizard.  @dgutride, we are not taking in breaking changes right now correct?

Reply:
Wizard header was not exported before so not a breaking change